README.md

```diff
@@ -147,7 +157,16 @@ $ pwd
 $ make
 ```
 
-## Step 6: Run the example experiment
+If you also want to try the example Java benchmarks, you must build them
+as a separate step:
+
+```bash
+$ pwd
+.../krun/examples/benchmarks
+$ make java_bench
```

Comment:
Got caught out by this. Typically, non-filename targets in makefiles use hyphens instead of underscores.